src/android/registerNativeModule.js

```diff
@@ -2,16 +2,14 @@ const readFile = require('./fs').readFile;
 const writeFile = require('./fs').writeFile;
 const compose = require('lodash').flowRight;
 const getReactVersion = require('../getReactNativeVersion');
```

Comment:
Do we actually use that function anywhere else? If not - I think we can remove it + remove `semver` dependency :)